### PR TITLE
quickstart: Filling gaps in the SEV documentation.

### PR DESCRIPTION
**Pull request by UnmeshDeodhar**
Fixing a couple of permission issues and command line parameters for skopeo.

Signed-off-by: Unmesh Deodhar <udeodhar@amd.com>

### CODE REVIEW

**Review by ryansavino — APPROVED** [review]
LGTM

**Comment by ryansavino** [comment]
@fitzthum @c3d @bpradipt

File: quickstart.md [hunk 1]
```diff
@@ -513,6 +513,7 @@ pod_ip=$(kubectl get pod -o wide | grep encrypted-image-tests | awk '{print $6;}
 Get the CoCo sample encrypted container image SSH access key from docker image label and save it to a file:
 
 ```
+sudo docker pull quay.io/kata-containers/encrypted-image-tests:encrypted
```
Comment (fitzthum) [inline quickstart.md:516]: While we are here, seems like we could (should?) be doing `docker` without `sudo`?

File: quickstart.md [hunk 2]
```diff
@@ -698,7 +699,7 @@ Run skopeo to encrypt the image created in the previous section:
 
 ```
 sudo OCICRYPT_KEYPROVIDER_CONFIG=$(pwd)/attestation-agent/sample_keyprovider/src/enc_mods/offline_fs_kbs/ocicrypt.conf \
-skopeo copy docker-daemon:[REGISTRY_URL]:unencrypted \
+skopeo copy --insecure-policy docker-daemon:[REGISTRY_URL]:unencrypted \
```
Comment (fitzthum) [inline quickstart.md:702]: Might be good to add a note about what `--insecure-policy` is. Don't want to frighten anyone with the i-word.

**Review by c3d — APPROVED** [review]
Thank you @UnmeshDeodhar, I am a bit sick since late last week, so only reviewing very quickly, but LGTM (with @fitzthum's comments addressed)

File: quickstart.md (hunk 2 above)
Comment (UnmeshDeodhar) [reply quickstart.md:702]: Added a note after the command.

**Review by fitzthum — APPROVED** [review]
Cool. Thanks @UnmeshDeodhar

**Pull request merged**